azure-pipelines.yml

```diff
@@ -68,11 +72,36 @@ jobs:
       set -e
 
       eval $(opam env)
-      ./configure -local -warn-error yes -native-compiler no
+      ./configure -prefix '$(Build.BinariesDirectory)' -warn-error yes -native-compiler no -coqide opt
       make -j "$NJOBS"
     displayName: 'Build Coq'
 
   - script: |
       eval $(opam env)
-      make -j "$NJOBS" test-suite
+      make -j "$NJOBS" test-suite PRINT_LOGS=1
     displayName: 'Run Coq Test Suite'
+
+  - script: |
+      make install
+    displayName: 'Install Coq'
+
+  - script: |
+      set -e
+      eval $(opam env)
+      export PKG_CONFIG_PATH=/usr/local/opt/libffi/lib/pkgconfig
+      VERSION=$(sed -n -e '/^let coq_version/ s/^[^"]*"\([^"]*\)"$/\1/p' configure.ml)
+      APP=bin/CoqIDE_${VERSION}.app
+      make PRIVATEBINARIES="$APP" "$APP"
+      make OLDROOT="$(Build.BinariesDirectory)" COQINSTALLPREFIX="$APP/Contents/Resources/" install-coq install-ide-toploop
+      mkdir -p "$DMGDIR"
+      ln -s /Applications "$DMGDIR/Applications"
+      cp -r "$APP" "$DMGDIR"
+      hdiutil create -imagekey zlib-level=9 -volname "coq-$VERSION-installer-macos" -srcfolder "$DMGDIR" -ov -format UDZO "$(Build.ArtifactStagingDirectory)/coq-$VERSION-installer-macos.dmg"
+    displayName: 'Create the dmg bundle'
```

Comment:
This script is a bit big, maybe put it in its own file (`azure-make-dmg.sh`?)

Reply:
Done. Thanks.